test/models/resume_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'

```

Comment:
this part still needs work. however, resume creation was tested comprehensivly using integration tests.